.github/dependabot.yml

```diff
@@ -1,5 +1,10 @@
 version: 2
 updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
why do we need to run it daily? gradle one is scheduled on weekly basis

Reply:
Oversight, changed it back to weekly